src/libbson/src/bson/bson-atomic.h

```diff
@@ -73,6 +73,12 @@ enum bson_memory_order {
 #define BSON_EMULATE_INT
 #endif
 
+/* CDRIVER-4264 Contrary to documentation, VS 2013 targeting x86 does not
+ * correctly/consistently provide _IntrinsicPointerExchange. */
```

Comment:
```suggestion
 * correctly/consistently provide _InterlockedExchangePointer. */
```

Reply:
Thank you. I keep mistakening "Interlocked" as "Intrinsic". Will fix. 😑